src/components/work-package-summary/work-package-summary.test.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { render, screen } from '@testing-library/react';
+import WorkPackageSummary from './work-package-summary';
+import { exampleWorkPackage1, exampleWorkPackage2, exampleWorkPackage3, WorkPackage } from 'utils';
+import { dollarsPipe, wbsPipe, listPipe, endDatePipe } from '../../shared/pipes';
+
+describe('Rendering Work Packagae Summary Test', () => {
+  test('Rendering example 1', () => {
+    const wp: WorkPackage = exampleWorkPackage1;
+    render(<WorkPackageSummary workPackage={wp} />);
+    expect(screen.getByText(`${wp.name}`)).toBeInTheDocument();
+    expect(screen.getByText(`${wbsPipe(wp.wbsNum)}`)).toBeInTheDocument();
+    expect(screen.getByText(`${wp.duration} weeks`)).toBeInTheDocument();
+
+    expect(screen.getByText(`${wp.deliverable}`)).toBeInTheDocument();
+    expect(
+      screen.getByText(`Dependencies:${listPipe(wp.dependencies, wbsPipe)}`)
+    ).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        `${listPipe(wp.rules, (str: string): string => {
+          return str;
+        })}`
```

Comment:
```suggestion
        `${listPipe(wp.rules, (str: string) => str)}`
```
Nice spot to use implicit return. I believe you can also remove the explicit return type because you're just returning the parameter so the type system can infer the return type really easily. I would recommend using this method to shorten the identity function in the rest of your files since it's so simple.